task-sdk/src/airflow/sdk/api/datamodels/_generated.py

```diff
@@ -118,21 +112,13 @@ class DagRunStateResponse(BaseModel):
 
 
 class DagRunType(str, Enum):
-    """
```

Comment:
Is this also related to some issue in cadwyn?

Reply:
Reported as https://github.com/zmievsa/cadwyn/issues/263